.github/workflows/ci.yml

```diff
@@ -28,6 +28,10 @@ jobs:
       - name: Test in Linux GraalVM
         if: matrix.os == 'ubuntu-latest' && matrix.java == 'graalvm'
         run: docker compose -f docker/Linux-GraalVM-JDK21/compose.yaml up --build --exit-code-from cantaloupe
+     # Only test one java version on Jammy   
+      - name: Test in Ubuntu Jammy
+        if: matrix.os == 'ubuntu-22.04' && matrix.java == 'jdk21'
+        run: docker compose -f docker/ubuntu/jammy/compose.yaml up --build --exit-code-from cantaloupe
```

Comment:
You added `noble/compose.yaml`, but it's not referenced in this file. Was that intentional.